tools/python/transit/gtfs/download_gtfs.py

```diff
@@ -17,17 +18,42 @@
 MAX_RETRIES = 2
 MAX_SLEEP_TIMEOUT_S = 30
 
+RAW_FILE_MOBILITYDB = "raw_mobilitydb.csv"
+
 URLS_FILE_TRANSITLAND = "feed_urls_transitland.txt"
-URLS_FILE_OMD = "feed_urls_openmobilitydata.txt"
+URLS_FILE_MOBILITYDB = "feed_urls_mobilitydb.txt"
+
+URL_MOBILITYDB_GTFS_SOURCE = "https://bit.ly/catalogs-csv"
 
 THREADS_COUNT = 2
 MAX_INDEX_LEN = 4
 
-HEADERS_OMD = {"Accept": "application/json"}
 
 logger = logging.getLogger(__name__)
 
 
+def download_gtfs_sources_mobilitydb(path):
+    """Downloads the csv catalogue from Data Mobility"""
+    try:
+        req = requests.get(URL_MOBILITYDB_GTFS_SOURCE)
+        url_content = req.content
+        csv_file = open(os.path.join(path, RAW_FILE_MOBILITYDB), 'wb')
+        csv_file.write(url_content)
+        csv_file.close()
```

Comment:
1. Will it close the file properly if write() above throws an exception?
2. After running this function, there will be a file somewhere in the file system. Is that place predictable and always the same? When should this file be deleted? Or it shouldn't? Does it make sense to reuse it if the HTTP connection fails?

Reply:
1. I have improved it. It should close now even if the write() fails.
2. The file can be reused even if the HTTP connection fails. The file is stored in the same location where the existing Transit Land URLs file is stored. It can be deleted but if it exists it will overwrite its content, so it will not cause any issues either way.